### DISCUSSION TITLE
wiki/Tutorials

### DISCUSSION THREAD

**Discussion by giscus** [gama-platform/gama-platform.github.io#185] [General]
# wiki/Tutorials

We propose some tutorials that are designed to allow modelers to become progressively autonomous with the GAMA platform. These tutorials cover different aspects of GAMA (Grid environment, GIS integration, 3D, multi-level modeling, equation-based models...). It is a good idea to keep a copy of the reference of the GAML language around when undertaking one of these tutorials.

https://gama-platform.org/wiki/Tutorials

**Comment by lesquoyb** [comment]
:+1:

**Comment by 19879099190** [comment]
can I ask you a question? It is how to calculate the distance from the settlement (SHP) to the nearest road (SHP) in GAMA
